### PR TITLE
feat(jitsi): create, send, receive and open jitsi links

### PR DESCRIPTION
**Pull request by gembarrett**
Please read complete the checklist, then fill out any sections that are relevant. Omit sections that are not relevant for cleanliness. 

### PR Checklist

- [x] I have merged `dev` into my branch before submitting
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Breaking changes

If this is a breaking change (remove section otherwise):

- [x] I have taken care to advise desktop devs or create a corresponding PR on peerio-desktop if applicable
- [x] I have taken care to advise mobile devs or create a corresponding PR on peerio-mobile if applicable

### Relevant user stories or bugs
User story 6321

### Gotchas/additional notes for reviewer
Related PR in mobile: https://github.com/PeerioTechnologies/peerio-mobile/pull/51 This branch feat-jitsi-dialog works alongside peerio-mobile branch feat-jitsi-button. Note from @flohdot: "all should wait to be merged until the full feature is ready and all PRs approved 🙂"

------

The repo owner is @anri-asaturov.

### CODE REVIEW

**Comment by anri-asaturov** [comment]
I can't merge this, it contains file folders. @seavan @gembarrett @flohbot

File: src/copy/en.json [hunk 1]
```diff
@@ -604,10 +604,7 @@
   "title_messageInfo": "Message info",
   "title_videoCallLink": "Use Jitsi for encrypted video chat",
   "title_videoCall": "Setup Jitsi Meet",
-  "dialog_videoCall": "Jitsi Meet is a web-based encrypted video chat and screen sharing service",
-  "disclaimer_videoCall": "Jitsi app required on mobile devices.",
-  "button_gotJitsi": "Got it!",
-  "button_notGotJitsi": "Get app",
+  "dialog_videoCall": "Jitsi Meet is a web-based encrypted video chat and screen sharing service.",
```
Comment (seavan) [inline src/copy/en.json]: if you look to the left, you see that your branch removes disclaimer_videoCall, button_gotJitsi, button_notGotJitsi. is this the intent?

Comment (gembarrett) [reply src/copy/en.json]: Yes, that dialog copy was changed to just a Dismiss button

**Comment by anri-asaturov** [comment]
Tests are failing but that's e2e timeouts, i need to increase them. Merging.

**Pull request merged**